auth0_flutter/README.md

```diff
@@ -293,36 +323,126 @@ You can make users land directly on the Signup page instead of the Login page by
+
+```dart
+final credentials =
+          await auth0.webAuthentication(useCredentialsManager: false).login();
 ```
 
 #### Web Auth errors
 
-Web Auth will only throw `WebAuthException` exceptions.<!-- Check the API documentation [link to API documentation] to learn more about the available `WebAuthException` properties. -->
+Web Auth will only throw `WebAuthException` exceptions. Check the [API documentation](https://pub.dev/documentation/auth0_flutter_platform_interface/latest/auth0_flutter_platform_interface/WebAuthException-class.html) to learn more about the available `WebAuthException` properties.
 
 ```dart
 try {
-  final result = await auth0.webAuthentication.login();
+  final credentials = await auth0.webAuthentication().login();
   // ...
 } on WebAuthException catch (e) {
-  print(e.toString())
+  print(e.toString());
+}
+```
+
+### Credentials Manager
+
+<!-- **See all the available features in the API documentation ↗** [link to API documentation] -->
+
+- [Check for stored credentials](#check-for-stored-credentials)
+- [Retrieve stored credentials](#retrieve-stored-credentials)
+- [Local authentication](#local-authentication)
+- [Credentials Manager errors](#credentials-manager-errors)
+
+The Credentials Manager utility allows you to securely store and retrieve the user's credentials. The credentials will be stored encrypted in Shared Preferences on Android, and in the Keychain on iOS. 
+
+> 💡 If you're using Web Auth, you do not need to manually store the credentials after login and delete them after logout; auth0_flutter does it automatically.
+
+#### Check for stored credentials
+
+When the users open your app, check for valid credentials. If they exist, you can retrieve them and redirect the users to the app's main flow without any additional login steps.
+
+```dart
+final isLoggedIn = await auth0
+        .webAuthentication()
+        .credentialsManager
+        ?.hasValidCredentials() ??
+    false;
+
+if (isLoggedIn) {
+  // Retrieve the credentials and redirect to the main flow
+} else {
+  // No valid credentials exist, present the login page
+}
+```
+
+#### Retrieve stored credentials 
+
+The credentials will be automatically renewed (if expired) using the [refresh token](https://auth0.com/docs/secure/tokens/refresh-tokens). **This method is thread-safe.**
+
+```dart
+final credentials =
+        await auth0.webAuthentication().credentialsManager?.credentials();
+```
+
+> 💡 You do not need to call `credentialsManager?.storeCredentials()` afterward. The Credentials Manager automatically persists the renewed credentials.
+
+#### Local authentication
+
+You can enable an additional level of user authentication before retrieving credentials using the local authentication supported by the device, for example PIN or fingerprint on Android, and Face ID or Touch ID on iOS.
+
+```dart
+final localAuthentication =
```

Comment:
The constructor of `LocalAuthenticationOptions` is not a `const` constructor, unlike the `IdTokenValidationConfig` one.